docs/csharp/language-reference/builtin-types/built-in-types.md

```diff
@@ -22,11 +21,15 @@ The following table lists the C# built-in [value](value-types.md) types:
 |[`float`](floating-point-numeric-types.md)|<xref:System.Single?displayProperty=nameWithType>|
 |[`int`](integral-numeric-types.md)|<xref:System.Int32?displayProperty=nameWithType>|
 |[`uint`](integral-numeric-types.md)|<xref:System.UInt32?displayProperty=nameWithType>|
+|[`nint`](integral-numeric-types.md)|<xref:System.IntPtr?displayProperty=nameWithType> \*|
+|[`nuint`](integral-numeric-types.md)|<xref:System.UIntPtr?displayProperty=nameWithType> \*|
```

Comment:
Why not to reference the article about native integers directly here?

Reply:
Good idea, I'll do that.

Reply:
Ah, sorry @tdykstra I was not clear. I've meant the links in the left-most column.